src/pages/Home.vue

```diff
@@ -1,6 +1,122 @@
 <template>
   <main>
-    <h1>LevelDB Inspector</h1>
-    <img src="@images/logo.png" alt="App Logo" />
+    <header>
+      <div class="logo"></div>
+    </header>
+    <form>
+      <p>Select path to LevelDB</p>
+      <label class="file db-path">
+        <input type="file" id="file" aria-label="File browser example" />
+        <span class="file-custom"></span>
+      </label>
+      <input type="submit" value="Inspect" />
```

Comment:
@jjlumagbas I have no actual data to back this up, but from my observation no one really does this anymore hahaha 😝 Submit buttons using `<button>` are more favorable because they are more flexible in terms of styling (e.g. pseudo-elements, etc.) and also they can contain additional HTML as their children.

Reply:
Haha ok fine will change to button